packages/edit-site/src/components/routes/link.js

```diff
@@ -49,18 +50,17 @@ export function useLink( params = {}, state, shouldReplace = false ) {
 	};
 }
 
-export default function Link( {
-	params = {},
-	state,
-	replace: shouldReplace = false,
-	children,
-	...props
-} ) {
+export function UnforwardedLink(
+	{ params = {}, state, replace: shouldReplace = false, children, ...props },
+	ref
+) {
 	const { href, onClick } = useLink( params, state, shouldReplace );
 
 	return (
-		<a href={ href } onClick={ onClick } { ...props }>
+		<a ref={ ref } href={ href } onClick={ onClick } { ...props }>
 			{ children }
 		</a>
 	);
 }
+
+export default forwardRef( UnforwardedLink );
```

Comment:
This is necessary to allow it to be used by `Composite`.

Reply:
So this is needed in order for `title` field to work properly, right? What happens if the components used by other fields doesn't forward the ref? In the future these fields could come from extenders. Will they need to always do that too?

If yes, can we absorb this in our internal implementation?

Reply:
This is because of how [Ariakit requires custom components to be open for extension](https://ariakit.org/guide/composition#custom-components-must-be-open-for-extension). I'm not sure how we'd absorb it, to be honest.